lib/views/pay/pay_view.dart

```diff
@@ -93,11 +110,14 @@ class _PayViewState extends State<PayView> {
                 ),
               ),
               alignLabelWithHint: true,
-              contentPadding: const EdgeInsets.all(20),
-              labelText: 'Invoice',
-              hintText: 'invoice',
+              contentPadding: const EdgeInsets.all(30),
+              labelText: 'Invoice/ btc address',
+              hintText: 'Invoice/ btc address',
             ),
           ),
+          const SizedBox(
+            height: 20,
+          ),
```

Comment:
hey it is not better use `Space` and not a `SizeBox`? imho the Sizebox is an antippattern

Reply:
Sure

Reply:
I think this will not work hahah sorry